safetrace-app/Services/AppEnvironment.swift

```diff
@@ -1,7 +1,17 @@
+import Foundation
+
 struct AppEnvironment: Environment {
     var bluetoothPermissions: BluetoothPermissionsProviding = BluetoothPermissionsProvider()
     var notificationPermissions: NotificationPermissionsProviding = NotificationPermissionsProvider()
     var safeTrace: SafeTraceProviding = SafeTraceProvider()
     var citizen: CitizenProviding = CitizenProvider()
     var analytics: AnalyticsTracking = AnalyticsTracker()
+
+    init() {
+        if Bundle.main.bundleIdentifier == "org.ctzn.safetrace-dev" {
```

Comment:
why are you using this instead of `#if INTERNAL`?

Reply:
Oh true, I can do that instead. Forgot that internal builds use the `staging` build config instead of `release`, so they have the `INTERNAL` flag

Reply:
Addressed